Neos.ContentRepository.Core/Classes/Projection/ContentGraph/PropertyCollection.php

```diff
@@ -55,9 +55,11 @@ public function offsetGet($offset): mixed
     {
         if (!isset($this->deserializedPropertyValuesRuntimeCache[$offset])) {
             $serializedProperty = $this->serializedPropertyValues->getProperty($offset);
-            $this->deserializedPropertyValuesRuntimeCache[$offset] = $serializedProperty === null
-                ? null
-                : $this->propertyConverter->deserializePropertyValue($serializedProperty);
+            if ($serializedProperty === null) {
+                return null;
+            }
+            $this->deserializedPropertyValuesRuntimeCache[$offset] =
+                $this->propertyConverter->deserializePropertyValue($serializedProperty);
```

Comment:
Why disabling the runtime cache here for `null` values? Might not have that impact as there is no deserialization, but, there is no need to treat it differently.

Reply:
actually we wrote to the runtime cache before but never read, because we were using isset instead of array_key_exists `!isset($this->deserializedPropertyValuesRuntimeCache[$offset])` and i thought well null doesnt have to be cached ^^

Reply:
From weekly discussion: replace isset with array_key_exists in line 56 and keep caching null as it may be the result of deserialization of something not null.